flowfile_core/flowfile_core/flowfile/code_generator/custom_node_handlers.py

```
@@ -0,0 +1,183 @@
+            for param in params[1:]:
+                if param.annotation != inspect.Parameter.empty:
+                    param_type_str = str(param.annotation)
+                    if "LazyFrame" in param_type_str:
+                        needs_collect = False
+                        break
+                if param.name in type_hints:
+                    hint_str = str(type_hints[param.name])
+                    if "LazyFrame" in hint_str:
+                        needs_collect = False
+                        break
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)